### DISCUSSION TITLE
This dependency was not found: * v-calendar/lib/components/date-picker.umd

### DISCUSSION THREAD

**Discussion by sedoy107** [google/timesketch#2987] [Q&A, quick solutions, support] [answered]
Good day,

I'm trying to build TimeSketch UI (frintend-ng) on macOS M1 Version 13.4.1. I use VSCode and I have installed latest `node` using homebrew. Then did `yarn install` which went fine. When run `yarn run serve` I get this compile error:

> yarn run v1.22.21
> $ NODE_ENV=development NODE_OPTIONS=--openssl-legacy-provider vue-cli-service serve
>  INFO  Starting development server...
> 98% after emitting CopyPlugin
> 
>  ERROR  Failed to compile with 1 error                                                                                                                                                                                                     2:02:27 PM
> 
> This dependency was not found:
> 
> * v-calendar/lib/components/date-picker.umd in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/components/Explore/AddManualEvent.vue?vue&type=script&lang=js
> 
> To install it, you can run: npm install --save v-calendar/lib/components/date-picker.umd

I tried `npm install --save v-calendar/lib/components/date-picker.umd` which tried to find the specified path in current working dir where I didn't have it.

I also tried `npm --save install v-calendar` but I got the dependency conflict.

I was on `20231025` tag but same happens on `master` branch as well.

If anyone know how to fix, please share your experience.

Thanks,
Sergey

**Comment by jkppr** [comment, marked as answer]
Hi, a temporary workaround is pinning the `v-calendar` version to `2.4.1` in the [package.json](https://github.com/google/timesketch/blob/master/timesketch/frontend-ng/package.json#L23) file.

Issue for tracking with the `v-calendar` project: https://github.com/nathanreyes/v-calendar/issues/1392
